examples/src/memory/astradb.ts

```diff
@@ -0,0 +1,48 @@
+import { BufferMemory } from "langchain/memory";
```

Comment:
We're trying to do all new examples with LCEL - could you update this to be more like: 

https://github.com/langchain-ai/langchainjs/pull/4450/files#diff-65d27e9b05b811faae72376bd775edc281f6b37233130c14f38613089a580554R1